deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/CbsAuthorizationType.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.deps.transport.amqp;
+
+/**
+ * An enumeration of supported authorization methods for AMQP connections that use Claims Based Security links.
+ */
+public enum CbsAuthorizationType
```

Comment:
Intentionally putting this in the deps package in case we need to re-use it for device client scenarios

Reply:
Doesn't the amqp connection for device client currently use cbs links?